heaps/heap_sort.py

```diff
@@ -1,8 +1,20 @@
-
+from heapq import heappush, heappop
 
 def heap_sort(list):
```

Comment:
👀 Nice work, but it would be great to implement this using the `MinHeap` class you implement below!